runbenchmark.py

```diff
@@ -358,9 +360,19 @@
             args.mode,
         )
 
-    bench.setup(amlb.SetupMode[args.setup])
-    if args.setup != "only":
-        res = bench.run(args.task, args.fold)
+    try:
+        bench.setup(amlb.SetupMode[args.setup])
+    except StaleProcessError as e:
```

Comment:
I'll move to exception notes instead and/or revise this structure. The reason I did it this way is to communicate more clearly to the user with a final message what went wrong and how to solve it. I want to generally make errors easier to parse, as there are some issues opened that are completely solvable from the traceback, but users can't/don't try to parse those.